### PR TITLE
light improvements

### PR DESCRIPTION
**Pull request by FrancoisRegisDegott-eaton**
* light-improvements
Signed-off-by: Degott, Francois Regis <FrancoisRegisDegott@eaton.com>

### CODE REVIEW

File: lib/src/nut_agent.cc [hunk 1]
```diff
@@ -111,6 +111,8 @@ void NUTAgent::updateDeviceList()
 
 int NUTAgent::send(const std::string& subject, zmsg_t** message_p)
 {
+	if (zsys_interrupted) return -100;
```
Comment (ericclappier-eaton) [inline lib/src/nut_agent.cc]: ident

File: lib/src/nut_agent.cc [hunk 2]
```diff
@@ -127,6 +129,8 @@ int NUTAgent::send(const std::string& subject, zmsg_t** message_p)
 // MVY: a hack for inventory messages
 int NUTAgent::isend(const std::string& subject, zmsg_t** message_p)
 {
+	if (zsys_interrupted) return -100;
```
Comment (ericclappier-eaton) [inline lib/src/nut_agent.cc]: ident

**Review by ericclappier-eaton — DISMISSED** [review]

File: lib/src/nut_agent.cc (hunk 1 above)
Comment (FrancoisRegisDegott-eaton) [reply lib/src/nut_agent.cc]: done... ;)

Comment (aquette) [reply lib/src/nut_agent.cc]: > ident

Indent 😉😁

**Review by aquette — APPROVED** [review]

**Pull request merged**